proposed/openebs/latest/questions.yml

```diff
@@ -56,7 +58,7 @@ questions:
     type: string
     label: Node Disk Manager Image
   - variable: ndm.imageTag
-    default: "v0.2.0"
+    default: "v0.3.0"
```

Comment:
@prateekpandey14 Can you update this one to 0.3.1 please?

Reply:
Done !